flink-core/src/main/java/org/apache/flink/api/common/functions/util/AbstractRuntimeUDFContext.java

```diff
@@ -83,10 +86,27 @@ public AbstractRuntimeUDFContext(
     }
 
     @Override
+    // The @Override annotation will be removed in FLINK-2.0 version to avoid exposing the
+    // executionConfig, and the getter method should only be used internally.
```

Comment:
And it's better to place the comment above overall method, include the annotation.
And if to keep the comment, it's better to state that "This method is for internal use. It will be still needed after the public interface is removed." in case someone spare efforts to remove this method in the future.